pkg/sbombastic-image-vulnerability-scanner/data/sbombastic.rancher.io.cveDetails.js

```diff
@@ -0,0 +1,302 @@
+export const cveDetail = {
+    "id": "CVE-2017-5337",
+    "severity": "critical",
+    "description": "The ReadMATImageV4 function in ImageMagick 7.0.8-7 uses an uninitialized variable, leading to memory corruption.",
+    "score": "9.9",
+    "cvssVersion": "v3",
+    "advisoryVendors": [
+      {
+        "name": "Microsoft",
+        "link": "https://docs.microsoft.com/en-us/security-updates/securitybulletins/2015/ms15-131"
+      },
+      {
+        "name": "SUSE",
+        "link": "https://www.suse.com/security/cve/CVE-2017-5337/"
+      },
+      {
+        "name": "Red Hat",
+        "link": "https://access.redhat.com/security/cve/CVE-2017-5337"
+      },
+      {
+        "name": "Alpine",
+        "link": "https://security.alpinelinux.org/vuln/CVE-2017-5337"
+      },
+      {
+        "name": "CISA",
+        "link": "https://www.cisa.gov/known-exploited-vulnerabilities-catalog/CVE-2017-5337"
+      }
+    ],
+    "cvssScores": [
+      {
+        "source": "NVD CVSSv3",
+        "score": 9.9,
+        "link": "https://nvd.nist.gov/vuln-metrics/cvss/v3-calculator?name=CVE-2017-5337"
+      },
+      {
+        "source": "NVD CVSSv2",
+        "score": 9.8,
+        "link": "https://nvd.nist.gov/vuln-metrics/cvss/v2-calculator?name=CVE-2017-5337"
+      }
+    ],
+    "affectedImages": 150,
+    "totalImages": 900,
+    "imagesIdentifiedDate": "Sep 01, 2025 10:00 AM",
+    "publishedDate": "Oct 19, 2024 4:37 AM",
+    "lastModifiedDate": "Sep 01, 2025 9:00 AM",
+    "sources": [
+      {
+        "name": "NVD",
+        "link": "https://nvd.nist.gov/vuln/detail/CVE-2017-5337"
+      },
+      {
+        "name": "GHSA",
+        "link": "https://github.com/advisories/GHSA-1234-5678"
+      }
+    ],
+    "references": [
```

Comment:
Since the references are associated with a particular advisory vendor as shown in the Figma design, I think this should be a subfield under `advisoryVendors[x].references`
<img width="759" height="567" alt="Screenshot 2025-09-05 at 4 23 52 PM" src="https://github.com/user-attachments/assets/4befbdce-6297-4e76-a51c-7e90985b665b" />